pkg/convert/tf.go

```diff
@@ -2606,6 +2611,8 @@ func translateRemoteModule(
 		sourceRoot, "/",
 		destinationRoot, destinationDirectory,
 		info,
+		requiredProviders,
+		false,
```

Comment:
Not sure how consistent we are in this repo but in `pu/pu` we often annotate "naked" parameters with their names e.g. `false, /*topLevelModule*/`.